tests/test_library_commands.py

```
@@ -0,0 +1,373 @@
+            self._mock_fetch_index,
+        ):
+            result = library_list(provider="invalid")
+            assert result == 1
+            captured = capsys.readouterr()
+            assert "No evaluators found for provider" in captured.out
+
+
+class TestLibraryInstall:
+    """Tests for library install command."""
+
+    def _mock_fetch_index(self, *args, **kwargs):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_mock_fetch_index`

(ANN202)